project/Dependencies.scala

```diff
@@ -16,69 +16,34 @@ import Keys._
 object Dependencies {
 
   val resolutionRepos = Seq(
-    "Sonatype" at "https://oss.sonatype.org/content/repositories/releases",
-    // For scala-util
-    "Snowplow Analytics" at "http://maven.snplow.com/releases/",
     // For Twitter's LRU cache
-    "Twitter Maven Repo" at "http://maven.twttr.com/",
-    // For scalaz-7.0 & specs2
-    "Scalaz Bintray Repo" at "http://dl.bintray.com/scalaz/releases"
+    "Twitter Maven Repo" at "http://maven.twttr.com/"
   )
 
   object V {
     // Java
-    val jodaTime      = "2.9.1"
-    val jodaConvert   = "1.8.1"
-
+    val jodaTime    = "2.9.9"
+    val jodaConvert = "1.8.2"
     // Scala
-    val scalaz        = "7.0.6"
-    val json4s        = "3.2.11"
-    val akka          = "2.3.14"
-    val akkaStreams   = "2.0.2"
-
-    object collUtil {
-      val _210      = "6.3.4"
-      val _211      = "6.29.0"  // Should work on 2.10 too
-    }
-
+    val json4s   = "3.2.11"
+    val scalaj   = "2.3.0"
+    val collUtil = "6.34.0"
```

Comment:
If we can bump `collUtil` to `6.39.0` then we will be able to publish scala-weather for Scala 2.12. Json4s for 2.12 has been published few days ago https://github.com/json4s/json4s/releases/tag/v3.2.11_2.12